app/src/test/java/com/github/sdpcoachme/ExampleUnitTest.kt

```diff
@@ -14,4 +16,16 @@ class ExampleUnitTest {
     fun addition_isCorrect() {
         assertEquals(4, 2 + 2)
     }
+
+    @Test
+    fun colorTest() {
```

Comment:
Does this test needs to be merged into production ?

Reply:
No, you are right, I deleted my test as it was mainly for me.